backend/inspirehep/matcher/parsers.py

```diff
@@ -68,9 +68,14 @@ def journal_title(self):
 
     @property
     def page_start(self):
-        return self.root.xpath(
+        page_start = self.root.xpath(
             "/biblStruct/monogr/imprint/biblScope[@unit='page']/@from"
         ).get()
+        if page_start:
+            return page_start
+        return self.root.xpath(
+            "/biblStruct/monogr/imprint/biblScope[@unit='page']/text()"
```

Comment:
alternatively you can select both at the XPath level (only one of those should exist, and you're picking that one with `.get()`):
```suggestion
        page_start = self.root.xpath(
            "/biblStruct/monogr/imprint/biblScope[@unit='page']/(@from | text())"
        ).get()
```

Reply:
@michamos nice one!

Reply:
though: `ValueError: XPath error: Invalid expression in /biblStruct/monogr/imprint/biblScope[@unit='page']/(@from | text())`

Reply:
`(/biblStruct/monogr/imprint/biblScope[@unit='page']/@from | /biblStruct/monogr/imprint/biblScope[@unit='page']/text())` this seems ok

Reply:
strange, here it works: https://www.freeformatter.com/xpath-tester.html